samples/nrf9160/modem_shell/src/uart/uart.c

```diff
@@ -38,4 +52,6 @@ void enable_uarts(void)
 	if (uart_dev) {
 		pm_device_state_set(uart_dev, PM_DEVICE_STATE_ACTIVE, NULL, NULL);
 	}
+
+	printk("UARTs enabled.\n");
```

Comment:
Not sure the dot is needed. Please check if there are other prints with the ending dot.
And check all prints in this PR.

Reply:
Do we need to use `printk()` in this file or rather `shell_print` etc.?

Reply:
Apparently shell_print cannot be used directly from timer handler. Keeping it as it is, like discussed.

Reply:
What about removing the dot as suggested by @trantanen?

Reply:
Should be fixed now.

Reply:
AFAIR timer handlers are executed directly from ISR context, so `shell_print()` cannot be used as it uses "ISR forbidden" kernel calls like `k_mutex_lock()`. If it's important for you to use `shell_print()` you may consider using system workqueue and a delayed work (https://github.com/zephyrproject-rtos/zephyr/blob/main/include/kernel.h#L3255).

Reply:
Thanks for clarification. I am not planning to change this.